src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -2390,6 +2397,15 @@ public function attributesToArray()
 	{
 		$attributes = $this->getArrayableAttributes();
 
+		// If an attribute is listed as to "encrypt", we'll decrypt it using Crypt facade.
+		foreach ($attributes as $key => $value)
+		{
+			if (in_array($key, $this->encrypt) && $value)
+			{
+				$attributes[$key] = \Crypt::decrypt($value);
```

Comment:
They're just simple accessors for instances stored in the IoC container, and are only supposed to make application code easier. They're not used anywhere in Laravel's core.